### PR TITLE
Add support files for OOP in Java vs Python article

### PR DESCRIPTION
**Pull request by JFincher42**
Files to support the OOP in Java vs Python article

### CODE REVIEW

**Comment by dbader** [comment]
@JFincher42 Let me know when this is ready to merge :-)

File: oop-in-java-vs-python/Car.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * Car class
+ * 
+ * Used for the Real Python OOP in Java vs Python article
+ * 
+ * Requires both Vehicle.java and Device.java
+ */
+
+public class Car extends Vehicle implements Device {
+
+    private int year; // Year of our ccar
```
Comment (dbader) [inline oop-in-java-vs-python/Car.java]: typo: ccar

Comment (JFincher42) [reply oop-in-java-vs-python/Car.java]: Corrected

File: oop-in-java-vs-python/car.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    @voltage.setter
+    def voltage(self, volts):
+        """Warn the user before resetting the voltage."""
+        print("Warning: this can cause problems!")
+        self._voltage = volts
+
+    @voltage.deleter
+    def voltage(self):
+        """Warn the user beore deleting the voltage."""
```
Comment (dbader) [inline oop-in-java-vs-python/car.py]: beore -> before ;)

File: oop-in-java-vs-python/car.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    def __eq__(self, other):
+        """Do these objects have the same year?"""
+        return self.year == other.year
+
+    def __lt__(self, other):
+        """Which object was made earlier than the other?"""
+        return self.year < other.year
+
+    def __add__(self, other):
+        """Add the objects together in our oredefined way."""
```
Comment (dbader) [inline oop-in-java-vs-python/car.py]: "oredefined"

**Comment by dbader** [comment]
Whoops found a bunch more, might make sense to crosscheck if we have the same typos in the code on the article

File: oop-in-java-vs-python/car.py (hunk 3 above)
Comment (JFincher42) [reply oop-in-java-vs-python/car.py]: Fixed those and a few others I saw as well.

**Comment by JFincher42** [comment]
Did a spell check on the comments, found one more in Car.java as well, but I think we've got them all.

**Pull request merged**